packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -125,12 +126,12 @@ final performanceOverlay = ToggleableServiceExtensionDescription<bool>._(
 
 final profileWidgetBuilds = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.profileWidgetBuilds',
-  description: 'Track Widget Rebuilds',
```

Comment:
Could the integration tests be failing because of this change?  Do they look for the description of the service extension buttons?